src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -1,5 +1,10 @@
 .. :changelog:
 
+unreleased
+----------
+
+* Allow equals sign in an environment variable value.
```

Comment:
Recommend this be more specific:  "container create: Fixes issue where equals sign was not allowed inside an environment variable.